app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,11 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+    <% if quote.created_at %>
```

Comment:
OH sure, look at http://guides.rubyonrails.org/active_record_validations.html

And add a "non null" constraint to the the  created_at field. Additionally, it looks like the dev data is seeded by the `production.sql` file -- change it so that the two records have some date -- maybe just use todays.